pipeline_templates/build_and_run_tests.yml

```diff
@@ -115,6 +123,21 @@ jobs:
         arguments: '-C "${{ parameters.build_configuration }}" --output-on-failure'
         workingFolder: 'cmake_build'
 
+  - ${{ if and(eq(parameters.FSANITIZE_TYPE, 'OFF'), and(eq(parameters.run_test_suite_without_ctest, ''), parameters.with_app_verifier)) }}:
```

Comment:
>run_test_suite_without_ctest [](http://example.com/codeflow?start=71&length=28)

note: for now I'm just adding this for the basic case of "run all tests in ctest". It can be expanded later for other `run_test_suite_without_ctest` cases, but I didn't need that for zrpc.